cart_app/views.py

```diff
@@ -1 +1,134 @@
+import json
 
+from django.views           import View
+from django.http            import (
+    JsonResponse,
+    HttpResponse
+)
+from django.core.exceptions import (
+    ObjectDoesNotExist,
+    ValidationError,
+    FieldDoesNotExist
+)
+
+from users.decorator        import login_check
+from users.models           import (
+    User,
+    UserProductColor
+)
+from product_app.models     import (
+    Size,
+    Product,
+    Color,
+    ProductColor,
+    ProductColorSize,
+    DetailImage
+)
+from cart_app.models        import (
+    Order,
+    OrderStatus,
+    Cart
+)
+
+class OrderView(View):
+    @login_check
+    def post(self, request):
+        try:
+            user    = request.user
+            user_id = user.id
```

Comment:
user를 사용하는 코드가 없어서 아래와 같이 줄여볼수 있을것같아요
```suggestion
            user_id = request.user.id
```